addons/test_mail/tests/test_mail_followers_unsubscribe.py

```diff
@@ -46,3 +53,148 @@ def test_inbox_notification_follower(self):
         self.assertEqual(follower.res_id, test_record.id)
         self.assertEqual(follower.partner_id, self.partner_employee)
         test_record.with_user(self.user_admin).message_unsubscribe(partner_ids=self.partner_employee.ids)
+
+
+@tagged('mail_followers', 'post_install', '-at_install')
+class UnfollowFromEmailTest(TestMessagePostCommon, HttpCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_record = cls.env['mail.test.simple'].browse(cls.test_record.ids)
+        cls.test_record_admin = cls.test_record.with_user(cls.user_admin)
+        cls.test_record_unfollow = cls.env['mail.test.simple.unfollow'].create({'name': 'unfollow'})
+        cls.test_record_unfollow_admin = cls.test_record_unfollow.with_user(cls.user_admin)
+        cls.user_employee.write({'notification_type': 'email'})
+        cls.partner_admin = cls.user_admin.partner_id
```

Comment:
Note that it is already done in mail common :)

Reply:
Thanks indeed, I have removed it.